### PR TITLE
Members only features

### PR DESCRIPTION
**Pull request by Maha-Magdy**
Hello,

This pull request for the project Members-only has been made by using the Rails app. I have used device gem. Devise helps me to authenticate users and authorize some views depends if they are logged in or not.

Please, review it, and tell me your opinion.

Best reagds,
Maha Magdy

### CODE REVIEW

**Review by enaburekhan — COMMENTED** [review]
# Status: Invalid
**Hi Maha!
Friendly reminder. Kindly make the `development` branch the default branch. And open the PR from the `members-only-features` to the development branch and not the main branch. This Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready.**

Do have questions/comments? kindly drop them :point_down: and ping me on slack @ Eric Enaburekhan to respond.
Happy hacking:blush:
Eric Enaburekhan.

**Review by MkrtichSargsyan — CHANGES_REQUESTED** [review]
## _Status:_ Changes Requested :recycle:

## Hello Maha Abdelaal 

THIS IS VERY IMPRESSIVE WORK. YOU HAVE DONE A GOOD JOB :+1:

## To Highlight :tada:

1. Project works as expected
2. Your PR has a good title and description.
3. You have descriptive git commit messages.


However, there are some changes that you need to make in order to get your project approved

- [x] Make the development branch the default branch

- [x] No need for the ```name``` field, when the user signs in

- [ ] Change the project title ```RE-FORMER _ Bare Metal Forms and Helpers project ```  in your Readme file

<hr>

## OPTIONAL

- [ ] There should be NO logic in views. Views are only for displaying information.
View-related logic can be encapsulated in View helpers 


Please, make the REQUIRED changes and submit your request again! :grinning:
Good luck. 

> ps: . If you have any questions about my review or need help, you can message me on slack ( @mkrtichsargsyan )

File: README.md [hunk 1]
```diff
@@ -1,24 +1,56 @@
...
-* Database initialization
-
-* How to run the test suite
-
-* Services (job queues, cache servers, search engines, etc.)
-
-* Deployment instructions
-
-* ...
+# RE-FORMER _ Bare Metal Forms and Helpers project
```
Comment (MkrtichSargsyan) [inline README.md]: Please change the project title

**Review by taiwo2 — APPROVED** [review]
# STATUS: APPROVED
@Maha-Magdy 
Thanks for implementing suggested changes.
kindly merge and prepare for next challenges ahead

**Pull request merged**